go/vt/vttablet/onlineddl/executor.go

```diff
@@ -4003,16 +4003,23 @@ func (e *Executor) updateDDLAction(ctx context.Context, uuid string, actionStr s
 
 func (e *Executor) updateMigrationMessage(ctx context.Context, uuid string, message string) error {
 	log.Infof("updateMigrationMessage: uuid=%s, message=%s", uuid, message)
-	query, err := sqlparser.ParseAndBind(sqlUpdateMessage,
-		sqltypes.StringBindVariable(message),
-		sqltypes.StringBindVariable(uuid),
-	)
-	if err != nil {
+
+	update := func(message string) error {
+		message = strings.ToValidUTF8(message, "")
```

Comment:
Do we want to use `�` as the replacement here since that's the canonical UTF-8 replacement character (and also what a console / log today would already print). 

See also https://www.fileformat.info/info/unicode/char/fffd/index.htm

Reply:
@dbussink sounds good. I'm down for maintenance, would you like to push that change?

Reply:
done